modules/@apostrophecms/area/index.js

```diff
@@ -554,7 +554,8 @@ module.exports = {
         const widgetEditors = {};
         const widgetManagers = {};
         const widgetIsContextual = {};
-        const contextualWidgetDefaultData = {};
+        const widgetHasInitialModal = {};
+        const contextualWidgetDefaultData = {}; // TODO: remove?
```

Comment:
**defaultData**

Seems like there's already a [default data mechanism](https://v3.docs.apostrophecms.org/reference/modules/widget-type.html#defaultdata) for widgets.

Is it something we want to keep?
I'm having difficulties to see in which context it is used.

Reply:
I was thinking more about the default content for areas that we are going to implement according to the Widget Placeholders tech design.

But that's something different I reckon.